web-ui/src/main/resources/catalog/components/search/searchmanager/SearchFormDirective.js

```diff
@@ -123,9 +123,9 @@
 
       $scope.searching++;
       $scope.searchObj.params = angular.extend({},
-          $scope.searchObj.defaultParams || defaultParams,
```

Comment:
It looks to be used in the directory entry selector and onlinesrc directive. eg. https://github.com/geonetwork/core-geonetwork/blob/develop/web-ui/src/main/resources/catalog/components/edit/directoryentryselector/partials/directoryentrylistselector.html#L21. Did you check those ?